docs/api.md

```diff
@@ -2474,7 +2474,362 @@ HTTP/1.1 200 Success
 
 ## EventAction
 
-> TODO
+`EventAction` groups emergency response actitvities
```

Comment:
Please update to be: Define an emergency response activity

Reply:
Done